ci/build_container/build_recipes/luajit.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+VERSION=2.0.5
```

Comment:
Suggest moving to 2.1-beta. OpenResty reported 40% speedups for its CDN workloads

Reply:
I would prefer to stay on the release version for now. There are going to be bugs in the rest of the code and I would prefer to not be worrying about bugs in the runtime.

Reply:
Commenting well after the fact, was it on your radar that luajit is functionally abandonware?  (https://www.freelists.org/post/luajit/Looking-for-new-LuaJIT-maintainers) I'm I'm not sure if that changes your opinion any.

Given it's been abandoned for years I'm not convinced 2.1 will ever make it out of beta.  Google's been using 2.1 (AFIK exclusively) since 2015 so I'd hope the kinks are pretty much worked out.  It turns out the generated lua in Envoy tests isn't 100% compatible with 2.1 which means we have a preference for upgrading upstream to 2.1 as well. :-)

Reply:
Yeah, I am aware of the situation. It's not completely abandoned. They are actually working on shipping 2.1. Sure, I can upgrade to 2.1 and fix whatever issues show up.

Reply:
@alyssawilk what error are you seeing exactly? I kind of suspect the issue is you aren't compiling internally w/ 5.2 compat options: https://github.com/envoyproxy/envoy/blob/master/ci/build_container/build_recipes/luajit.sh#L35

You probably are going to have to build LuaJIT twice with different defines. :(